### DISCUSSION TITLE
About the bug of tool jwt-generator

### DISCUSSION THREAD

**Discussion by XiaLuoHun** [he3-app/he3-feedback#178] [Bug Report]
### Version
jwt-generator 1.0.3

### Environment
Windows v1.3.7

### What is actually happening?

输入的json键值对和预览结果不对等
如输入键值对为
{
  "exp": "1684207705.823902"
}, 

预览键值对为
{
    "exp": 1684207705
}

左侧输入后必须切换到KV界面,然后点添加才可以到达预览框, 比较麻烦, 建议左侧输入json键值对后,直接在预览结果界面显示

**Comment by lyzhang1999** [comment]
感谢您的建议，我已经创建了 issues：https://github.com/he3-app/he3-feedback/issues/179，您可以跟进该 issues 的状态。
